src/storage/src/hummock/sstable/builder.rs

```diff
@@ -165,7 +177,22 @@ impl SSTableBuilder {
             version: VERSION,
         };
 
-        (self.buf.freeze(), meta)
+        (
+            self.buf.freeze(),
+            meta,
+            if self.vnode_bitmaps.len() > SST_BITMAP_LIMIT {
+                vec![]
```

Comment:
`SST_BITMAP_LIMIT` is 0 and we will also reach here to produce empty bitmap

Reply:
fixed. now `SST_BITMAP_LIMIT` == 10